tests/helpers.py

```diff
@@ -609,6 +609,41 @@ def create_multiple_pvcs(
     ]
 
 
+def create_restore_pvc(
```

Comment:
please move this function to `ocs_ci/ocs/resources/pvc.py`

Reply:
Fixed